net/api/test.cc

```diff
@@ -135,13 +136,12 @@ class UseLocalHTTPTestServer {
       HTTPServerConnection connection(socket.Accept());
       const auto& message = connection.Message();
       const string method = message.Method();
-      const string url = message.URL();
+      const URL url = URL(message.Path());
       if (method == "GET") {
-        if (url == "/get") {
-          connection.SendHTTPResponse("DIMA");
-        } else if (url == "/drip?numbytes=7") {
+        if (url.path == "/drip") {
+          const size_t numbytes = atoi(url["numbytes"].c_str());
```

Comment:
The indexer on the `url` looks nice at first sight, but it would be more logical from an API standpoint to put the parsed query parameters into a separate `query` property, e.g. `url.query["numbytes"]`.